nansat/exporter.py

```diff
@@ -254,7 +264,8 @@ def export2thredds(self, filename, bands={}, metadata=None, mask_name=None, no_m
         data.export(tmp_filename, rm_metadata=rm_metadata)
         del data
 
-        self._post_proc_thredds(tmp_filename, filename, bands, dstBands, time, global_metadata)
+        self._post_proc_thredds(
```

Comment:
It seems like a crucial last step inside the method, please add a comment line before that which gives a hint on what you are going for.

Reply:
Done